tests/pin_project.rs

```diff
@@ -295,30 +291,30 @@ fn lifetime_project() {
 
     impl<T, U> Struct<T, U> {
         fn get_pin_mut<'a>(self: Pin<&'a mut Self>) -> Pin<&'a mut T> {
-            self.project_into().pinned
+            self.project().pinned
```

Comment:
I didn't see this as a problem because I know both ways, but it would be easier to do all of this with a single method.